boards/espressif/esp32_ethernet_kit/Kconfig.defconfig

```diff
@@ -27,3 +25,18 @@ config HEAP_MEM_POOL_ADD_SIZE_BOARD
 choice BT_HCI_BUS_TYPE
 	default BT_ESP32 if BT
 endchoice
+
+endif # BOARD_ESP32_ETHERNET_KIT_ESP32_PROCPU
+
+if BOARD_ESP32_ETHERNET_KIT_ESP32_APPCPU
+
+config SOC_ESP32_APPCPU
+	default y
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	default 4096
+
+config KERNEL_BIN_NAME
+	default "esp32_appcpu_firmware"
```

Comment:
why is this being set?

Reply:
This comes from V1 model. Sample `ipm_esp32` requires `remote` build to be added in main sample as .c source:  
```c
set_source_files_properties(${REMOTE_ZEPHYR_DIR}/esp32_appcpu_firmware.c PROPERTIES GENERATED TRUE)
target_sources(app PRIVATE src/main.c ${REMOTE_ZEPHYR_DIR}/esp32_appcpu_firmware.c)
```
That sample needs to be converted to sysbuild. Can we keep as is for now?

Reply:
This needs to go, neither builds or specific boards should be including a .c file, it can be added back in once it is properly using sysbuild. No board with this is going to pass CI

Reply:
Then that sample will not work until it is updated. Are you ok with that?